src/Components/DrillIllustration.js

```diff
@@ -90,7 +90,7 @@ const DrillIllustration = props => {
   const displayAnimation = ({ illustrationSource, instruction, title }) => {
     return (
       <>
-        <Animation readonly widthRatio={1} heightRatio={1 / 2} animation={illustrationSource} />
```

Comment:
I considered an Animation is readonly by default, and must be tagged as editable to be used in the editor